.travis.yml

```diff
@@ -0,0 +1,42 @@
+# references:
+# * http://www.objc.io/issue-6/travis-ci.html
+# * https://github.com/supermarin/xcpretty#usage
+
+language: objective-c
+
+# cache: cocoapods
+podfile: Example/Podfile
+
+before_install:
+  - brew install swiftlint
+
+osx_image:
+  - xcode7.3
+  - xcode8
+# env:
+#   matrix:
+#     - TASK=build
+#     - TASK=lintpod
+
+matrix:
+    include:
+        - os: osx
+          osx_image: xcode8
+          env:
+            - TASK=build
+              DESTINATION="platform=iOS Simulator,name=iPhone 6,OS=10.0"
+        - os: osx
+          osx_image: xcode7.3
+          env:
+            - TASK=build
+              DESTINATION="platform=iOS Simulator,name=iPhone 6,OS=9.3"
+        - os: osx
+          osx_image: xcode7.3
+          env: TASK=lintpod
+        # - os: osx
+        #   osx_image: xcode8
+        #   env: TASK=lintpod
```

Comment:
Looks like the trouble I had with linting using Xcode 8 may be resolved now in CocoaPods 1.1.0.beta.1 CocoaPods/CocoaPods#5529